src/smart-components/Channel/components/ChannelUI/index.tsx

```diff
@@ -152,7 +153,9 @@ const ChannelUI: React.FC<ChannelUIProps> = ({
       <div className="sendbird-conversation__footer">
         {
           renderMessageInput?.() || (
-            <MessageInput />
+            <MessageInput
+              ref={messageInputRef}
```

Comment:
Maybe lets add  messageInputRef to channelProvider?

Reply:
Yeah it would be better